Tests/Azure/test_azure_config.py

```diff
@@ -0,0 +1,12 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import pytest
+
+from InnerEye.Azure.azure_config import AzureConfig
+
+
+def test_validate() -> None:
+    with pytest.raises(ValueError):
```

Comment:
you should check the error message here, to ensure that it does not fail validation for another reason